### ISSUE TITLE
The /planters/{planterId} page

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-web-map-client#132]
* Implement the /planters/{id} page.
* Write integration test to cover the page, check the [guide](https://github.com/Greenstand/treetracker-web-map-client#test) of int test. 
* The UI: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=2497%3A11031
* Prerequites: a common component in this page, check video below.

https://www.loom.com/share/547c1e48753648739ad0dc7a376ded65

![image](https://user-images.githubusercontent.com/5744708/133953243-e07ce3b5-c9ac-4344-9dca-6e37aa3f7775.png)

---
Some hints:
* Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
* Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
* To know more about our organization, visit our [website](https://greenstand.org).

**Comment by dadiorchen** [comment]
This issue now is open for everyone

**Comment by ojafero** [comment]
I'm almost done fixing the time and location component. So, will get working on this asap.

**Comment by jupiterdoom** [comment]
is this issue still actual?

**Comment by dadiorchen** [comment]
Yes, this is actual the most important task for us

**Comment by dadiorchen** [comment]
Do you want to take it?

**Comment by jupiterdoom** [comment]
the issue looks to be fully/nearly completed
![image](https://user-images.githubusercontent.com/11030075/146668787-f8cccdea-4de5-481e-86c6-d681565a599d.png)

**Comment by dadiorchen** [comment]
Some guys have helped to integrate parts of this page, yes, the page looks fine now, the only thing left is the integration test, we need cover this page by integration cypress test, if you are interested, you can write test for this, if not interested, it's also fine, please take a look others, here are some of them:
https://github.com/Greenstand/treetracker-web-map-client/issues/304
https://github.com/Greenstand/treetracker-web-map-client/issues/334
There are more in the project page:
https://github.com/orgs/Greenstand/projects/26
https://github.com/orgs/Greenstand/projects/28

In short, just do as your wish :)

**Comment by jupiterdoom** [comment]
@dadiorchen thanks, i would take #334. Can you add me in slack: jupiterdoom@gmail.com

**Comment by pierrelstan** [comment]
@dadiorchen, I can work on this?

**Comment by dadiorchen** [comment]
Sure!

**Comment by pierrelstan** [comment]
> Sure!

Thank you
